src/libraries/System.Memory/tests/Span/IndexOfAnyValues.cs

```diff
@@ -13,6 +15,59 @@ namespace System.SpanTests
 {
     public static partial class SpanTests
     {
+        private static readonly Func<IndexOfAnyValues<byte>, byte[]> s_getValuesByteMethod =
```

Comment:
i guess you're intentionally getting it from `IndexOfAnyValues<T>` rather than `IndexOfAnyValuesInRange<T>`, as future refactorings might introduce new subclasses that also should work.

Reply:
Oh, never mind I mis-grepped. But this made me notice you're missing "" to exercise IndexOfEmptyValues.

Reply:
Added the case for empty values as well